### ISSUE TITLE
downloading https://ghcr.io/v2/linuxbrew/core/XXXX/manifests failure

### ISSUE THREAD

**Issue by zrma** [Homebrew/linuxbrew-core#23168]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
HOMEBREW_VERSION: 3.1.5-14-ga81a76b
ORIGIN: https://github.com/Homebrew/brew
HEAD: a81a76b196e987210a0f1f2dcfe8453483a23549
Last commit: 7 hours ago
Core tap ORIGIN: https://github.com/Homebrew/linuxbrew-core
Core tap HEAD: 1b335ee000c1ca1c2a52edb7dfc9f46c989952d2
Core tap last commit: 6 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.3 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: octa-core 64-bit skylake
Clang: 12.0 build (parse error)
Git: 2.25.1 => /usr/bin/git
Curl: 7.68.0 => /usr/bin/curl
Kernel: Linux 5.4.72-microsoft-standard-WSL2 x86_64 GNU/Linux
OS: Ubuntu 20.04.2 LTS (focal)
Host glibc: 2.31
/usr/bin/gcc: 9.3.0
/usr/bin/ruby: N/A
glibc: N/A
gcc@5: 5.5.0_6
xorg: N/A
```

```plaintext
Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  isl@0.18

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /home/bulbitain/.pyenv/shims/python-config
  /home/bulbitain/.pyenv/shims/python3-config
  /home/bulbitain/.pyenv/shims/python3.8-confi
```

### 

- [X] I ran `brew update` and am still able to reproduce my issue.
- [ ] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

I tried to upgrade some programs.

```plaintext
brew update && brew upgrade
Already up-to-date.
==> Upgrading 3 outdated packages:
erlang 23.3.1 -> 23.3.2
awscli 2.1.39 -> 2.2.0
libdrm 2.4.104 -> 2.4.105
==> Upgrading erlang 23.3.1 -> 23.3.2
==> Downloading https://ghcr.io/v2/linuxbrew/core/libdrm/manifests/2.4.105
#=#=#                                                                     
##O#- #                                                                   
##O=#  #                                                                  
#=#=-#  #                                                                 
curl: (22) The requested URL returned error: 404

Error: Failed to download resource "libdrm_bottle_manifest"
Download failed: https://ghcr.io/v2/linuxbrew/core/libdrm/manifests/2.4.105
==> Upgrading awscli 2.1.39 -> 2.2.0
==> Downloading https://ghcr.io/v2/linuxbrew/core/libdrm/manifests/2.4.105
#=#=#                                                                     
##O#- #                                                                   
##O=#  #                                                                  
curl: (22) The requested URL returned error: 404

Error: Failed to download resource "libdrm_bottle_manifest"
Download failed: https://ghcr.io/v2/linuxbrew/core/libdrm/manifests/2.4.105
==> Upgrading libdrm 2.4.104 -> 2.4.105
==> Downloading https://ghcr.io/v2/linuxbrew/core/libdrm/manifests/2.4.105
#=#=#                                                                     
##O#- #                                                                   
##O=#  #                                                                  
curl: (22) The requested URL returned error: 404

Error: Failed to download resource "libdrm_bottle_manifest"
Download failed: https://ghcr.io/v2/linuxbrew/core/libdrm/manifests/2.4.105
```

### What happened (include all command output)?

failed to download resource

### What did you expect to happen?

success to upgrade

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
$ brew update && brew upgrade
```

I tried to connect `https://ghcr.io/v2/linuxbrew/core/XXXX/manifests` directly, and it returned like that.
```json
{"errors":[{"code":"UNAUTHORIZED","message":"authentication required"}]}
```

thank you for all your help.

**Comment by iMichka** [comment]
Fixed by 523ca76086ab4979042f9294bf261d507f7548a7

**Comment by zrma** [comment]
> Fixed by [523ca76](https://github.com/Homebrew/linuxbrew-core/commit/523ca76086ab4979042f9294bf261d507f7548a7)

I checked it works well.
Thanks for all your helps.
